growth/too/tasks/scheduler.py

```diff
@@ -157,8 +162,12 @@ def schedule_ztf(json_data):
                            'validity_window_mjd':
                                json_data["validity_window_mjd"],
                            'queue_type': 'list'})
-    r.raise_for_status()
 
+    if (r.status_code == 200) or (r.status_code == 400):
+        flash(r.text, 'danger')
+    elif r.status_code == 201:
+        flash(r.text, 'success')
+    log.exception(r.text)
```

Comment:
`log.exception()` should be used inside an `except:` block.

Reply:
I don't quite understand... what should this be?

Reply:
```python
try:
    r.raise_for_status()
except HTTPError:
    log.exception('submission to scheduler failed (%s)', r.text)
    flash(r.text)
else:
    flash(r.text, success)
```